### DISCUSSION TITLE
A Conjuntivite do Sonic

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#291] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/tirinhas/a-conjuntivite-do-sonic/

**Comment by angellisy** [comment]
Amei as expressões em italiano nas falas do Mario!

**Comment by one-eyed-willy** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [quarta-feira, 13 de abril de 2022](<https://www.vish.blog.br/imagens/tirinhas/a-conjuntivite-do-sonic/#comment_1098543477>):

> Amei as expressões em italiano nas falas do Mario!

Aderiu bem aos estereótipos linguísticos que a gente imagina pro personagem.
